content/components/only-manipulate-the-dom-via-the-renderer.md

```diff
@@ -0,0 +1,50 @@
+---
+title: only manipulate the DOM via the Renderer
+author: 
+  name: Billy Lando (@billyjov)
```

Comment:
Do you mind removing your handle here and only keep your name? I am just asking this to be consistent across all items.

Reply:
Maybe we can add another field for Twitter, if some people would like to add this information to the item as well.